plugins/performance-lab/includes/admin/load.php

```diff
@@ -92,11 +92,25 @@ function perflab_get_dismissed_admin_pointer_ids(): array {
  */
 function perflab_get_admin_pointers(): array {
 	$pointers = array(
-		'perflab-admin-pointer'            => __( 'You can now test upcoming WordPress performance features.', 'performance-lab' ),
-		'perflab-feature-view-transitions' => __( 'New <strong>View Transitions</strong> feature now available.', 'performance-lab' ),
-		'perflab-feature-nocache-bfcache'  => __( 'New <strong>No-cache BFCache</strong> feature now available.', 'performance-lab' ),
+		'perflab-admin-pointer' => __( 'You can now test upcoming WordPress performance features.', 'performance-lab' ),
 	);
 
+	// Get installed plugins to check if these feature plugins are already installed.
+	$installed_plugin_slugs = array_map(
+		static function ( $name ) {
+			return strtok( $name, '/' );
+		},
+		array_keys( get_plugins() )
+	);
+
+	// Only show if the feature plugin is not already active and not installed.
+	if ( ! in_array( 'view-transitions', $installed_plugin_slugs, true ) ) {
```

Comment:
Similar to `SPECULATION_RULES_VERSION`, we should also check for `VIEW_TRANSITIONS_VERSION` here.

The same applies to the **No-cache BFCache** plugin, but I noticed it doesn’t currently define a version: [https://github.com/westonruter/nocache-bfcache/blob/main/nocache-bfcache.php](https://github.com/westonruter/nocache-bfcache/blob/main/nocache-bfcache.php).

@westonruter Do you think we should define a version there as well, similar to how other Performance Lab plugins handle it?

Reply:
It does define a version, actually: https://github.com/westonruter/nocache-bfcache/blob/1f7118afee2dbdc43fc1642a7eac4a8ab81911c5/nocache-bfcache.php#L37

This constant is being used currently by the PL plugin when constructing the meta generator tag.

Reply:
@ShyamGadde I think what you've done makes sense. The admin pointer should be be shown if a feature plugin has been installed or activated, since either is a signal that the use is already aware of the feature and doesn't need the pointer to inform them of anything.